README.md

```diff
@@ -572,13 +585,13 @@ A promise automaton generated by the tableau construction of [`ltl2tgba`](http:/
 
 The product of a `Rabin 2` automaton with a `Streett 2` automaton, performed in a very straightforward way and representing the intersection of the two languages, will have an acceptance condition such as the following, where sets 0-3 are used as Rabin pairs, while sets 4-7 are used as Streett pairs.
 
-    Acceptance: 8 ((Inf(0)&Fin(1))|(Inf(2)&Fin(3)))&((Fin(4)|Inf(5))&(Fin(6)|Inf(7)))
+    Acceptance: 8 ((Fin(0)&Inf(1))|(Fin(2)&Inf(3))) & ((Inf(4)|Fin(5))&(Inf(6)|Fin(7)))
```

Comment:
The original example had the order wrong for `Rabin 2`.  We should remember to fix it even if this patch is rejected.